events.go

```diff
@@ -286,26 +286,26 @@ type EventDeploymentFailed struct {
 
 // EventDeploymentInfo describes a 'deployment_info' event.
 type EventDeploymentInfo struct {
-	EventType   string          `json:"eventType"`
-	CurrentStep *DeploymentStep `json:"currentStep"`
-	Timestamp   string          `json:"timestamp"`
-	Plan        *DeploymentPlan `json:"plan"`
+	EventType   string            `json:"eventType"`
+	CurrentStep *DeploymentStepV1 `json:"currentStep"`
```

Comment:
Renaming `DeploymentStepV1`to `CurrentStep` sounds like a good choice to me.

Just to be sure we're on the same page: I understand we'll use that struct in the event types you have mentioned and the deployments unmarshaling for Marathon version 1.x (i.e., [here](https://github.com/gambol99/go-marathon/blob/2e73c1d824745b3b6374f2d7cc46932f5a596698/deployment.go#L91)).